zerver/webhooks/zapier/view.py

```diff
@@ -1,15 +1,32 @@
-from typing import Any, Dict
+from typing import Any, Dict, List
 
 from django.http import HttpRequest, HttpResponse
 from django.utils.translation import ugettext as _
 
 from zerver.decorator import api_key_only_webhook_view
 from zerver.lib.actions import check_send_private_message_from_emails
 from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.stream_subscription import get_stream_subscriptions_for_user
 from zerver.lib.response import json_error, json_success
 from zerver.lib.webhooks.common import check_send_webhook_message, \
     validate_extract_webhook_http_header
-from zerver.models import UserProfile
+from zerver.models import UserProfile, get_stream_by_id_in_realm, \
+    Stream
+
+def get_available_streams_for_bot_owner(bot: UserProfile) -> List[str]:
+    assert bot.bot_owner is not None
+    subs = get_stream_subscriptions_for_user(bot.bot_owner)
```

Comment:
Could you just do something like `return get_stream_subscriptions_for_user(bot.bot_owner).filter(active=True).values_list('name', flat=True)` here? 
I don't know this code very well at all, so feel free to ignore if that doesn't make sense at all.

Reply:
Well, I don't think I can, no. `get_stream_subscriptions_for_user` returns a `QuerySet` of `Subscription` objects, which contains `Recipient` objects (which in turn contains the ID of the `Stream` object), so I can't really filter by the stream's attributes or anything because we start with a `Subscription` object. :)

Reply:
But I don't know this code too too well myself, so the probability of me being wrong is fairly high here. :)